ga4gh/serverconfig.py

```diff
@@ -45,6 +58,38 @@ class ProductionConfig(BaseConfig):
     DATA_SOURCE = None
 
 
+class GoogleOIDCConfig(ProductionConfig):
+    """
+    Configuration that is a good basis for production deployments using
+    Google as the authentication provider.
+    """
+    REQUEST_VALIDATION = True
+    # We should complain loudly if data source is not set, rather than
```

Comment:
Why is this key and comment repeated in each of these? We should set it once in the superclass.